data/provinces.json

```diff
@@ -0,0 +1,93 @@
+[
+    {
+        "name": {
+            "en": "Alberta",
+            "fr": "Alberta"
+        },
+        "short": "Alberta"
```

Comment:
again with the codes, leveraging the standards, `AB`, `BC`, ext.

Reply:
Same as language and month

Reply:
Just for the reflag. _We (Mohamad) build this API right? If so we should change it._